config/management/operational/src/governance.rs

```diff
@@ -40,6 +40,9 @@ impl AddValidator {
 
         // Verify that this is a configured validator
         client.validator_config(self.input.account_address)?;
+        let name = client
+            .validator_config(self.input.account_address)?
+            .human_name;
```

Comment:
Is there any reason you can't merge the two 'client.validator_config(..)?' calls? e.g., just remove the previous line 42 which calls the validator_config() method but does nothing with the return value?

Reply:
woops